packages/init/src/utils/create-repo.ts

```diff
@@ -1,30 +1,22 @@
 import type { Models } from "@slicemachine/core";
 import * as logs from "./logs";
-import * as Prismic from "@slicemachine/core/build/prismic";
+import { Client } from "./client";
 
 export function createRepository(
   domain: string,
-  framework: Models.Frameworks,
-  cookies: string,
-  base: string
+  framework: Models.Frameworks
 ): Promise<string> {
   const spinner = logs.spinner("Creating Prismic Repository");
   spinner.start();
 
-  return Prismic.Communication.createRepository(
-    domain,
-    cookies,
-    framework,
-    base
-  )
-    .then((res) => {
-      const addressUrl = new URL(base);
-      const repoDomainName = res.data.domain || domain;
-      addressUrl.hostname = `${repoDomainName}.${addressUrl.hostname}`;
+  return Client.createRepository(domain, framework)
+    .then((domain: string) => {
+      const addressUrl = new URL(Client.get().apisEndpoints.Wroom);
+      addressUrl.hostname = `${domain}.${addressUrl.hostname}`;
       const address = addressUrl.toString();
       spinner.succeed(`We created your new repository ${address}`);
 
-      return repoDomainName;
+      return domain;
```

Comment:
domain is declared at the top of this function ;)

Reply:
Which one, domain is defined on line 6 and 15 ;)
I guess they'll both be the same,